docs: add operator documentation (#3406)
inspektor-gadget/inspektor-gadget: docs/reference/operators/socketenricher.md

---
title: SocketEnricher
---

> The Socket enricher operator provides process information for sockets. Gadgets

Reviewer: ```suggestion
The Socket enricher operator provides process information of the socket owner. Gadgets
```
